pyccel/codegen/printing/cucode.py

```diff
@@ -87,6 +167,7 @@ def _print_ModuleHeader(self, expr):
                     }}\n'
         return '\n'.join((f"#ifndef {name.upper()}_H",
                           f"#define {name.upper()}_H",
+                          imports,
```

Comment:
Is this a bug fix? I think it should be mentioned in the PR description

Reply:
Yes, it is a bug that I found. I will fix it right now

Reply:
I mentioned that bug in the PR descriptio